libs/geoservices/covid/src/lib/pages/maps/components/time-map/time-map.component.html

```diff
@@ -0,0 +1,50 @@
+<div id="map-section">
+  <tamu-gisc-drawer [width]="'20rem'" [position]="'left'">
+    <div class="content">
+      <h3>Map Controls</h3>
+      <input type="checkbox" name="county-state-toggle" (click)="stateToggle()">
+      <label for="county-state-toggle">Counties/State</label>
+      <br>
+      <input type="checkbox" name="infection-mortality-toggle" (click)="mortalityToggle()">
+      <label for="county-state-toggle">Infection Rate/Mortality Rate</label>
+      <input type="date" id="datepicker" value="2020-06-06" max="2020-06-06" min="2020-01-21">
+      <tamu-gisc-button value="Register" type="submit"></tamu-gisc-button>
+    </div>
+  </tamu-gisc-drawer>
+  <tamu-gisc-mapbox-map
+    token="pk.eyJ1Ijoid2hhdHppdG9veWEiLCJhIjoiY2poNzI3M2M0MDNnbjMzcWVtYmZnNXExNiJ9.K-nM-JaZnZvAfutkLQitbg"
+    [center]="['-98', '38.88']">
+  </tamu-gisc-mapbox-map>
+  <div class='right-sidebar'>
+    <h3>COVID-19 Data</h3>
+    <div #infobox *ngIf="(infoBoxModel | async) as info">
+      <p *ngIf="!stateButtonToggle">County: {{info.name}}</p>
+      <p *ngIf="stateButtonToggle">State: {{info.name}}</p>
+      <p>Population: {{info.pop}}</p>
+      <p>Cases: {{info.cases}}</p>
+      <p>Infection Rate: {{info.infectionRate.toFixed(2)}}</p>
+      <p>Deaths: {{info.deaths}}</p>
+      <p>Mortality Rate: {{info.deathRate.toFixed(2)}}</p>
+    </div>
+    <div class="legend">
```

Comment:
Should this have an `NgIf` or does it show all the time, even when the mortality rate layer is enabled?

Reply:
When the mortality rate is enabled, this dissapears. I can add ngIf for consistency with the other legend.